### DISCUSSION TITLE
Performance benefits from specific components?

### DISCUSSION THREAD

**Discussion by devivekw** [buefy/buefy#3346] [Q&A] [answered]
Going through the [constructor options documenation](https://buefy.org/documentation/constructor-options) I see that it is possible to import specific components.

For example :
```js
// Constructor for Buefy 
import Vue from 'vue'
import { Table, Input } from 'buefy'

Vue.use(Table)
Vue.use(Input)
```

**Does this prove to have performance benefits or reduce bundle size for the app?** 
I am not really sure, fairly in experienced with this.

**Comment by jtommy** [comment, marked as answer]
Yes, in this way you can reduce the bundle size about js part
